mods/lord/lord_default/init.lua

```diff
@@ -20,17 +44,11 @@ minetest.register_craftitem(":default:charcoal_lump", {
 minetest.register_node(":default:charcoalblock", {
 	description       = S("Charcoal Block"),
 	tiles             = { "default_charcoal_block.png" },
-	is_ground_content = true,
+	is_ground_content = false,
 	groups            = { cracky = 3, flammable = 10 },
 	sounds            = default.node_sound_stone_defaults(),
-	on_punch          = function(pos, node, puncher)
-		if puncher:get_wielded_item():get_name() == "default:torch" then
-			local pos_above = { x = pos.x, y = pos.y + 1, z = pos.z }
-			if minetest.get_node(pos_above).name == "air" then
-				minetest.set_node(pos_above, { name = "fire:basic_flame" })
-			end
-		end
-	end,
+	on_punch          = fire_by_torch_punch,
+	_tt_help          = S('Ignited by a torch'),
```

Comment:
```suggestion
	_tt_help          = S("Ignited by a torch"),
```
но вообще на такое автоформаттер нужен

Reply:
спс